### PR TITLE
feat: [PL-29962]: UICore version upgrade: Fixes word-wrap issue and Avatar Component issues

### PR DESCRIPTION
**Pull request by mayankVermaHarness**
### Summary

JIRA: https://harness.atlassian.net/browse/PL-29962
UICore version upgrade: Fixes word-wrap issue and Avatar Component issues

UICore PR for Avatar component: https://github.com/harness/uicore/pull/926
UICore PR for word-wrap: https://github.com/harness/uicore/pull/930

#### Screenshots

**AVATAR COMPONENT**
**No Name, email or src**
image

**How it looks in Avatar Group**
Before
image

After (different sizes)
image
image
image
image
image

**WORD WRAP**
BEFORE:
image

AFTER:
image

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by avinashmadhwani02 — DISMISSED** [review]

**Comment by mayankVermaHarness** [comment]
retrigger cypress-pipeline

**Review by avinashmadhwani02 — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
     "@harness/monaco-yaml": "^1.0.0",
     "@harness/ng-tooltip": "^1.31.62",
     "@harness/telemetry": "^1.0.42",
-    "@harness/uicore": "^3.98.1",
+    "@harness/uicore": "^3.99.2",
```
Comment (knagurski) [inline package.json:65]: Why is this bumping the minor and patch? It looks like this is a new feature, so it should bump the minor version
```suggestion
    "@harness/uicore": "^3.99.0",
```

Comment (mayankVermaHarness) [inline package.json:65]: There were other changes as well in UICore that are further needed. So bumped to the latest version of uicore

Comment (knagurski) [reply package.json:65]: Sorry, I thought this was for the UI Core repo when I left my comment. This is fine here 👍🏻 Sorry for the confusion

**Pull request merged**